src/model/actions/quality-action.ts

```diff
@@ -29,7 +29,7 @@ export abstract class QualityAction extends GeneralAction {
         break;
     }
 
-    buffMod += (simulation.getBuff(Buff.INNER_QUIET)?.stacks || 0) / 10;
+    let iqMod = simulation.getBuff(Buff.INNER_QUIET)?.stacks || 0;
```

Comment:
Should be a const instead as we neevr edit it
```suggestion
    const iqMod = simulation.getBuff(Buff.INNER_QUIET)?.stacks || 0;
```

Reply:
Thanks! First time I do anything with typescript code, sorry!